maven/bnd-export-plugin/.gitignore

```diff
@@ -0,0 +1 @@
+/target/
```

Comment:
Ok .. will remove it .. I normally also do not check them in but I thought you had the policy to do so .. Probably confused this with openhab :-)